articles/quickstart/spa/_includes/_calling_api_create_scope.md

```diff
@@ -1,7 +1,11 @@
 ## Add a Scope
 
-When an `access_token` is received after a user authenticates in your app, it is simply an indication that they have proven their identity. By default, however, the `access_token` does not contain any **authorization** information. To properly limit access to your resources based on authorization, you must use [scopes](https://auth0.com/docs/api-auth/tutorials/adoption/api-tokens).
+By default, the access token does not contain any authorization information. To limit access to your resources based on authorization, you must use scopes. Read more about scopes in the  [api tokens documentation](https://auth0.com/docs/api-auth/tutorials/adoption/api-tokens).
```

Comment:
I left the link as is, but shouldn't it be just (api-auth/tutorials/adoption/api-tokens)?